test/Microsoft.AspNetCore.Mvc.IntegrationTests/FormCollectionModelBindingIntegrationTest.cs

```diff
@@ -25,7 +25,8 @@ private class Address
         {
             public int Zip { get; set; }
 
-            public IFormCollection FileCollection { get; set; }
+            [ModelBinder(Name = "Files")]
+            public IFormFileCollection FileCollection { get; set; }
```

Comment:
This completely changes the test. And the new test doesn't belong in the `FormCollectionModelBindingIntegrationTest`. `IFormFileCollection` is handled by `FormFileModelBinderProvider` and `FormFileModelBinder`.